src/msg/async/Stack.cc

```diff
@@ -122,19 +127,20 @@ Worker* NetworkStack::get_worker()
   // TODO: add more logic and heuristics, so connections known to be
   // of light workload (heartbeat service, etc.) won't overshadow
   // heavy workload (clients, etc).
-  if (!current_best || ((threads.size() < (unsigned)cct->_conf->ms_async_max_op_threads)
-      && (min_load > num_workers))) {
-     ldout(cct, 20) << __func__ << " creating worker" << dendl;
-     add_thread(num_workers);
-     spawn_workers(threads);
-     current_best = workers[num_workers++];
-     simple_spin_unlock(&pool_spin);
-     current_best->wait_for_init();
-  } else {
-    simple_spin_unlock(&pool_spin);
-    ldout(cct, 20) << __func__ << " picked " << current_best 
-                   << " as best worker with load " << min_load << dendl;
-  }
+  // if (!current_best || ((num_workers < (unsigned)cct->_conf->ms_async_max_op_threads)
```

Comment:
could you squash this commit into the commit where `NetworkStack` is introduced. and also remove the comment, if you please.